tests/unittests/AddressBook/MutableMultiValueTests.mm

```diff
@@ -0,0 +1,154 @@
+#import <Foundation/Foundation.h>
+
+#import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABAddressBook.h>
+#import <AddressBook/ABRecord.h>
+#import <AddressBook/ABPerson.h>
+#import <AddressBook/ABMultiValue.h>
+#import <AddressBook/ABMutableMultiValue.h>
+#import "ABContactInternal.h"
+
+void addPhone(WACContact* contact, WACContactPhoneKind kind, NSString* number) {
+    WACContactPhone* phone = [WACContactPhone make];
+    phone.number = number;
+    phone.kind = kind;
+
+    [contact.phones addObject:phone];
+    [phone release];
+}
+
+ABMultiValueRef createMultiValue() {
+    WACContact* contact = [WACContact make];
+    addPhone(contact, WACContactPhoneKindHome, @"0000000000");
+    addPhone(contact, WACContactPhoneKindMobile, @"1111111111");
+    ABRecordRef record = (__bridge_retained ABRecordRef)[[_ABContact alloc] initWithContact:contact];
+    [contact release];
+    ABMultiValueRef multiValue = (ABMultiValueRef)ABRecordCopyValue(record, kABPersonPhoneProperty);
+    CFRelease(record);
+    return multiValue;
+}
+
+TEST(AddressBook, MutableMultiValueCreateTest) {
+    ABMultiValueRef multiValue = createMultiValue();
+    ABMutableMultiValueRef mutableMultiValue = ABMultiValueCreateMutableCopy(multiValue);
+    ASSERT_EQ_MSG(ABMultiValueGetCount(multiValue),
+                  ABMultiValueGetCount(mutableMultiValue),
+                  "FAILED: The mutable multi value should have the same count!\n");
+    for (int i = 0; i < ABMultiValueGetCount(mutableMultiValue); i++) {
+        CFStringRef value = (CFStringRef)ABMultiValueCopyValueAtIndex(multiValue, i);
+        CFStringRef valueCopy = (CFStringRef)ABMultiValueCopyValueAtIndex(mutableMultiValue, i);
+        ASSERT_OBJCEQ_MSG((__bridge NSString*)value, (__bridge NSString*)valueCopy, "FAILED: Values should be the same!\n");
+        CFRelease(value);
+        CFRelease(valueCopy);
+
+        CFStringRef label = ABMultiValueCopyLabelAtIndex(multiValue, i);
+        CFStringRef labelCopy = ABMultiValueCopyLabelAtIndex(mutableMultiValue, i);
+        ASSERT_OBJCEQ_MSG((__bridge NSString*)label, (__bridge NSString*)labelCopy, "FAILED: Labels should be the same!\n");
+        CFRelease(label);
+        CFRelease(labelCopy);
+    }
+    CFRelease(mutableMultiValue);
+    CFRelease(multiValue);
+}
+
+TEST(AddressBook, MutableMultiValueAddAndReplaceTest) {
+    // Tests for adding label/value pairs.
+    ABMutableMultiValueRef multiValue = ABMultiValueCreateMutable(kABStringPropertyType);
+    ASSERT_EQ_MSG(0, ABMultiValueGetCount(multiValue), "FAILED: Count of multivalue should be 0!\n");
+    ABMultiValueIdentifier identifier1 = -1;
+    ABMultiValueIdentifier identifier2 = -1;
+    ASSERT_TRUE_MSG(ABMultiValueAddValueAndLabel(multiValue, @"0000000000", kABHomeLabel, &identifier1),
+                    "FAILED: Error adding first value/label pair!\n");
+    ASSERT_EQ_MSG(1, ABMultiValueGetCount(multiValue), "FAILED: Count of multivalue should be 1!\n");
```

Comment:
>"FAILED: Count of multivalue should be 1!\n [](start = 55, length = 43)

nit: don't need messages here, it will be clear from normal failure message